internal/app/machined/pkg/controllers/cri/seccomp_profile_test.go

```diff
@@ -123,9 +123,7 @@ func (suite *CRISeccompProfileSuite) TestReconcileSeccompProfile() {
 		},
 	})
 
-	ctest.UpdateWithConflicts(suite, cfg, func(mc *config.MachineConfig) error {
-		return nil
-	})
+	ctest.UpdateWithConflicts(suite, cfg, func(mc *config.MachineConfig) error { return nil })
```

Comment:
this is a no-op right, since it's not updating anything

Reply:
Yep - I think I was reformatting code and covered this too.